sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -132,6 +132,10 @@ class EventHubConsumerClient(
     :keyword uamqp_transport: Whether to use the `uamqp` library as the underlying transport. The default value is
      False and the Pure Python AMQP library will be used as the underlying transport.
     :paramtype uamqp_transport: bool
+    :keyword float socket_timeout: The time in seconds that the underlying socket on the connection should
+     wait when sending and receiving data before timing out. The default value is 0.2 for TransportType.Amqp
+     and 1 for TransportType.AmqpOverWebsocket. If connection errors are occurring due to write timing out,
```

Comment:
It might be worth specifying _what_ errors a customer would expect to see - i.e. is it an EventHubsConnectionError object? Is that how this issue would present?
Or would these never be raised in EH and the customer would just see more logging information?
The SDK structure is quite different between SB and EH, so not sure how this issue would present and how the API should reflect that.

Reply:
The user sees an `EventHubsConnectionError` similar to what was happening in SB. I was able to see this on send :)

Reply:
I've updated the comment and added in wording that Johan that suggested earlier that is for advanced usage scenarios